app/templates/script_ldap_query.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:proactive:jobdescriptor:3.7"
+        xsi:schemaLocation="urn:proactive:jobdescriptor:3.7 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.7/schedulerjob.xsd"
+        name="script_LDAP_Query"
+        priority="normal"
+        onTaskError="continueJobExecution"
+        maxNumberOfExecution="2"
+>
+    <taskFlow>
+        <task name="LDAP_Query_Task">
+            <description>
+                <![CDATA[ The task that makes query to a LDAP server ]]>
+            </description>
+            <variables>
+                <variable name="ldapUrl" value="ldap://localhost:389" inherited="false" model="String"/>
+                <variable name="ldapSearchBase" value="dc=yourOrganization,dc=com" inherited="false" model="String"/>
+                <variable name="ldapSearchFilter" value="(objectclass=*)" inherited="false" model="String"/>
+                <variable name="ldapSelectedAttributes" value="attributeName1,attributeName2" inherited="false" model="String"/>
+            </variables>
+            <scriptExecutable>
+                <script>
+                    <code language="groovy">
+                        <![CDATA[
+// Query a LDAP server requires authentication for connection
+// Please look at the following link for the configuration LDAP credentials:
```

Comment:
`for the configuration LDAP credentials` --> `for configuring LDAP credentials` or `for the configuration about LDAP credentials`?